index.qmd

```diff
@@ -0,0 +1,32 @@
+---
+title: "Southern California R Users Group"
+page-layout: full
+toc: false
+---
+
+```{r}
+#| include: false
+
+events <- readRDS("events/events_past_and_upcoming.rds")
+```
+
+Welcome to the Southern California R Users Group (SoCal RUG)! Since no single group can serve this sprawling area, several Los Angeles and Orange County groups joined forces to centralize SoCal-based R events and information.
```

Comment:
It is more than LA and OC. Consider rewording.

Reply:
Good point, thank you